content/articles/zone-files.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: Domain zone files
+excerpt: Welcome to DNSimple. This page is about zone files. Hosted DNS has never been this easy.

```

Comment:
:+1: 

Thought we had to leave the :hankey: for SEO purposes. Happy to change it :wink: !